tasks/section5.yml

```diff
@@ -31,15 +31,19 @@
       - rule_5.1.2
 
 - name: "SCORED | 5.1.2 | PATCH | Ensure permissions on /etc/crontab are configured"
-  command: /bin/true
+  file:
+      dest=/etc/crontab
+      owner=root
+      group=root
+      mode=0600
```

Comment:
This task needs to be in YAML literal form per [https://github.com/ansible/ansible-lockdown/blob/master/CONTRIBUTING.md](the style guide)